### ISSUE TITLE
composer init is failing without any reason

### ISSUE THREAD

**Issue by ejaaz-di** [composer/composer#9502]
Output of `composer diagnose`:

```Checking platform settings: OK
Checking git settings: OK
Checking http connectivity to packagist: OK
Checking https connectivity to packagist: OK
Checking github.com rate limit: OK
Checking disk free space: OK
Checking pubkeys:
Tags Public Key Fingerprint: 57815BA2 7E54DC31 7ECC7CC5 573090D0  87719BA6 8F3BB723 4E5D42D0 84A14642
Dev Public Key Fingerprint: 4AC45767 E5EC2265 2F0C1167 CBBB8A2B  0C708369 153E328C AD90147D AFE50952
OK
Checking composer version: OK
Composer version: 2.0.7
PHP version: 7.4.0
PHP binary path: C:\wamp64\bin\php\php7.4.0\php.exe
OpenSSL version: OpenSSL 1.1.1d  10 Sep 2019
cURL version: 7.67.0 libz 1.2.11 ssl OpenSSL/1.1.1d
zip extension: OK
```

When I run this command:

`composer init`

I get the following output:

```  Welcome to the Composer config generator



This command will guide you through creating your composer.json config.

Package name (<vendor>/<name>) [pc/packages]:

  [Symfony\Component\Console\Exception\RuntimeException]
  Aborted


init [--name NAME] [--description DESCRIPTION] [--author AUTHOR] [--type [TYPE]] [--homepage HOMEPAGE] [--require REQUIRE] [--require-dev REQUIRE-DEV] [-s|--stability STABILITY] [-l|--license LICENSE] [--repository REPOSITORY]
```

**Comment by stof** [comment]
PHP 7.4.0 has a bug on windows regarding reading from stdin (that's https://bugs.php.net/bug.php?id=78883). The bug has been fixed in PHP 7.4.1, so please upgrade your PHP version.

**Comment by ejaaz-di** [comment]
> PHP 7.4.0 has a bug on windows regarding reading from stdin (that's https://bugs.php.net/bug.php?id=78883). The bug has been fixed in PHP 7.4.1, so please upgrade your PHP version.

Thanks. @stof
